ngi_pipeline/engines/piper_ngi/local_process_tracking.py

```diff
@@ -358,14 +362,17 @@ def update_coverage_for_sample_seqruns(project_id, sample_id, piper_qc_dir,
     for libprep_id, seqruns in seqruns_by_libprep.iteritems():
         for seqrun_id in seqruns:
             label = "{}/{}/{}/{}".format(project_id, sample_id, libprep_id, seqrun_id)
+            genome_results_file_path=glob.glob(os.path.join(piper_qc_dir, "{}.{}*.qc".format(sample_id, seqrun_id.split('_')[-1]),"genome_results.txt"))[0]
```

Comment:
Well, I don't really understand how the folders are named. It looks like it is 
<sample>.<flowcell>.<sample>.<lane ?>.qc
I don't have lane data at that point, so I need to "guess" it with glob. If you have a better solution though, I'm all ears.